dev/requirements.txt

```diff
@@ -0,0 +1,4 @@
+boto3
```

Comment:
don't do a dev requirements file and a dev route in setup.py, I'd prefer just a dev route in setup.py

Reply:
Sure. The issue is that there are at least a couple of pkgs (`boto3`, `fs-miniofs`) that are required for the examples in the notebooks, but are not required for dev in general. Still, I guess it will be simplest to just put them all in the `[dev]` list in setup.py

Reply:
yeah im not super concerned about an overly-long dev dependency list

Reply:
done